### PR TITLE
Fix issue #171 (allow query() and where() in Query)

### PR DESCRIPTION
**Pull request by beowulfenator**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #171

### CODE REVIEW

File: QueryBuilder.php [hunk 1]
```diff
@@ -70,11 +70,19 @@ public function build($query)
...
         if ($whereQuery) {
-            $parts['query'] = $whereQuery;
-        } else if ($query->query) {
-            $parts['query'] = $query->query;
+            $conditionals[] = $whereQuery;
+        }
+        if ($query->query) {
+            $conditionals[] = $query->query;
+        }
+        if (count($conditionals) == 2) {
```
Comment (samdark) [inline QueryBuilder.php:82]:
```suggestion
        if (count($conditionals) === 2) {
```

File: QueryBuilder.php [hunk 2]
```diff
@@ -70,11 +70,19 @@ public function build($query)
...
+            $parts['query'] = ['bool' => ['must' => $conditionals]];
+        } elseif (count($conditionals) == 1) {
```
Comment (samdark) [inline QueryBuilder.php:84]:
```suggestion
        } elseif (count($conditionals) === 1) {
```

**Comment by beowulfenator** [comment]
Strict comparisons? No problem.

**Comment by samdark** [comment]
👍

**Pull request merged**